zephyr/module.yml

```diff
@@ -15,26 +15,26 @@ blobs:
     description: "Firmware for NXP IW612 Wi-Fi + Bluetooth single-chip solution"
     doc-url: https://www.nxp.com/products/wireless-connectivity/wi-fi-plus-bluetooth-plus-802-15-4/2-4-5-ghz-dual-band-1x1-wi-fi-6-802-11ax-plus-bluetooth-5-4-plus-802-15-4-tri-radio-solution:IW612
   - path: rw61x/rw61x_sb_ble_15d4_combo_a2.bin
-    sha256: 41e9837790c23c25b5e29defef87cbcc559f452c3bbcc4c894429237e3a2e98d
+    sha256: 7d94d41c4778b691292a325b881e972cb676467e57bfc804b0d6f8926d5a8dda
     type: img
     version: '1.0'
     license-path: zephyr/blobs/license/LA_OPT_NXP_Software_License.txt
-    url: https://github.com/nxp-mcuxpresso/mcux-sdk/raw/MCUX_2.15.000_RW610_K32W1/components/conn_fwloader/fw_bin/rw61x_sb_ble_15d4_combo_a2.bin
+    url: https://raw.githubusercontent.com/NXP/wifi_nb_fw/wifi_nb_fw_rel/rw61x_sb_ble_15d4_combo_a2.bin
```

Comment:
We should be pointing to a tag to a release and not getting the latest as the SHA will break whenever the binary in the target github is update.

Reply:
Hi @mmahadevan108  @dleach02 
Okay, how to point to a particular release/sha?

Reply:
Hi @mmahadevan108 @dleach02 please review again.